example/package.json

```diff
@@ -24,6 +24,7 @@
     "react-native-reanimated": "^2.5.0",
     "react-native-safe-area-context": "^4.2.5",
     "react-native-screens": "3.12.0",
+    "react-native-svg": "^13.8.0",
```

Comment:
Can we update with the latest version? 
https://github.com/software-mansion/react-native-svg/releases/tag/v13.11.0